LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any book obtaining a copy
```

Comment:
Changed the wrong files when using IntelliJ to change all occurrences of "person".

Reply:
😂Will fix that!!